compiler/one-cmds/CMakeLists.txt

```diff
@@ -101,7 +101,8 @@ set(ONE_PYTHON_FILES constant.py
                      OptionBuilder.py
                      TopologicalSortHelper.py
                      WorkflowRunner.py
-                     utils.py)
+                     utils.py
+                     O1.txt)
```

Comment:
1) this is not python file. please move to so somewhere else add new copy sections.

2) please do not use `.txt` file. this file format is text but not the content. maybe `.list` or `.options`.